modules/cli/src/main/scala/scala/cli/commands/Uninstall.scala

```diff
@@ -0,0 +1,49 @@
+package scala.cli.commands
+
+import caseapp._
+
+import scala.cli.CurrentParams
+import scala.cli.commands.util.VerbosityOptionsUtil._
+
+object Uninstall extends ScalaCommand[UninstallOptions] {
+  def run(options: UninstallOptions, args: RemainingArgs): Unit = {
+    CurrentParams.verbosity = options.bloopExit.logging.verbosityOptions.verbosity
+    val interactive =
+      options.bloopExit.logging.verbosityOptions.interactiveInstance(forceEnable = true)
+
+    val binDirPath =
+      options.binDirPath.getOrElse(scala.build.Directories.default().binRepoDir / "scala-cli")
+    val destBinPath = binDirPath / options.binaryName
+    val cacheDir    = scala.build.Directories.default().cacheDir
+
+    if (
+      !Update.isScalaCLIInstalledByInstallationScript() && (options.binDir == None || (options.binDir != None && !options.force))
+    ) {
```

Comment:
NIT The condition can be simplified a bit, and calling `isEmpty` on an option is more idiomatic than comparing it to `None` (and hopefully, scalafmt is fine having everything on the same line with those changes):
```suggestion
    if (!Update.isScalaCLIInstalledByInstallationScript() && (options.binDir.isEmpty || !options.force)) {
```